src/Symfony/Component/Validator/Constraints/Week.php

```diff
@@ -38,9 +38,9 @@ public function __construct(
         public ?string $min = null,
         public ?string $max = null,
         public string $invalidFormatMessage = 'This value does not represent a valid week in the ISO 8601 format.',
-        public string $invalidWeekNumberMessage = 'The week "{{ value }}" is not a valid week.',
-        public string $tooLowMessage = 'The value should not be before week "{{ min }}".',
-        public string $tooHighMessage = 'The value should not be after week "{{ max }}".',
+        public string $invalidWeekNumberMessage = '"{{ value }}" is not a valid week.',
```

Comment:
Actually, we don't have any other constraint that contain the placeholder value
```suggestion
        public string $invalidWeekNumberMessage = 'This value is not a valid week.',
```

Reply:
indeed 👍